.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,33 +1,26 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[BUG]"
+title: "A concise title here!"
 labels: bug
-assignees: tbaumer22
-
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior.
+Steps to reproduce the behaviour.
 
-**Expected behavior**
+**Expected behaviour**
 A clear and concise description of what you expected to happen.
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+**Supporting Information**
+If applicable, add screenshots, stacktraces or other information to help explain your problem.
```

Comment:
```suggestion
If applicable, add screenshots, stacktraces, or other information to help explain your problem.
```

Reply:
That makes no sense as an independent clause.

Reply:
It's an oxford comma; it has nothing to do with clauses

Reply:
:(